client/package.json

```diff
@@ -30,7 +31,8 @@
     "typescript-eslint": "^7.10.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start:local": "env-cmd -f environment/.env.local react-scripts start",
```

Comment:
These scripts are used to pull in the environment variables from the `*.env` files I've added.